department-of-veterans-affairs/vets-website
# [18164] Update Past Appointment Cards

## Description
This PR updates the past appointments styles on the VAOS homepage.

~Note: desktop styles for the date selector isn't defined in the designs. @outerpress~
UPDATE: The designs are updated to include the date dropdown styling.

To test:
- Ensure the `va_online_scheduling_homepage_refresh` feature flag is enabled
- Go the VAOS homepage [http://localhost:3001/health-care/schedule-view-va-appointments/appointments/](http://localhost:3001/health-care/schedule-view-va-appointments/appointments/)
- Choose `Past` from the dropdown
- Adjust the date range dropdown and validate output

## Testing done
Local and Unit

## Screenshots
![image](https://user-images.githubusercontent.com/8315447/105246969-054f8a00-5b42-11eb-9bad-b76a1df94d85.png)
![image](https://user-images.githubusercontent.com/8315447/105246665-94a86d80-5b41-11eb-8a2c-3aa21db734e4.png)
![image](https://user-images.githubusercontent.com/8315447/105246600-7e021680-5b41-11eb-9a68-5a18146586a0.png)
![image](https://user-images.githubusercontent.com/8315447/105246633-86f2e800-5b41-11eb-8cf4-5e68f8899b05.png)
![image](https://user-images.githubusercontent.com/8315447/105552554-13ccab80-5cd2-11eb-86c5-37f1ff81883e.png)

## Acceptance criteria
- [x] Changes are behind the va_online_scheduling_homepage_refresh feature flag
- [x] When VAOS homepage loads, default view is Upcoming appointments
- [x] When selecting Past from dropdown, display past appointments
- [x] DESIGN - All desktop, tablet, and mobile designs match the specs:
> Mobile - https://www.sketch.com/s/4198dca5-69c7-4769-9686-d213c900876d/a/wL9pyOm#Inspector
> Mobile template - https://www.sketch.com/s/4198dca5-69c7-4769-9686-d213c900876d/a/kaglvRD#Inspector
> Desktop/Tablet - https://www.sketch.com/s/4198dca5-69c7-4769-9686-d213c900876d/a/9PEAv3p#Inspector

## Definition of done
